lib/compressMeshes.js

```diff
@@ -0,0 +1,524 @@
+            const numIndices = indices.length;
+
+            console.log("Num of faces: " + numFaces);
+            meshBuilder.AddFacesToMesh(newMesh, numFaces, indices);
+
+            // Add attributes to mesh.
+            var attributes = primitive.attributes;
+            var attributeToId = {};
+            for (var semantic in attributes) {
+                const attributeData = getNamedAttributeData(gltf, primitive, semantic);
+                const numPoints = attributeData.numVertices;
+                var attributeName = semantic;
+                if (semantic.indexOf('_') !== -1)
+                  attributeName = attributeName.substring(0, semantic.indexOf('_'));
+
+                const data = new Float32Array(attributeData.data);
+                var attributeId = -1;
+                if (attributeName === 'POSITION' || attributeName === 'NORMAL' ||
+                    attributeName === 'COLOR' ) {
+                  attributeId = meshBuilder.AddFloatAttributeToMesh(newMesh, encoderModule[attributeName],
+                      numPoints, attributeData.numComponents, data);
+                } else if (semantic === 'TEXCOORD_0') {
+                  attributeId = meshBuilder.AddFloatAttributeToMesh(newMesh, encoderModule.TEX_COORD,
+                      numPoints, attributeData.numComponents, data);
+                } else {
+                  attributeId = meshBuilder.AddFloatAttributeToMesh(newMesh, encoderModule.GENERIC,
+                      numPoints, attributeData.numComponents, data);
+                }
+
+                if (attributeId === -1) {
+                  console.log("Error: Failed adding attribute " + semantic);
+                  return;
+                } else {
+                  attributeToId[semantic] = attributeId;
+                }
+            }
+
+            let encodedDracoDataArray = new encoderModule.DracoInt8Array();
+            encoder.SetSpeedOptions(10 - compressionLevel, 10 - compressionLevel);  // Compression level is 10 - speed.
+            if (useUnifiedQuantization) {
+              encoder.SetAttributeExplicitQuantization(encoderModule.POSITION, positionQuantization, 3, positionOrigin, positionRange);
+            } else {
+              encoder.SetAttributeQuantization(encoderModule.POSITION, positionQuantization);
+            }
+            encoder.SetAttributeQuantization(encoderModule.NORMAL, normalQuantization);
+            encoder.SetAttributeQuantization(encoderModule.TEX_COORD, texcoordQuantization);
+            encoder.SetAttributeQuantization(encoderModule.COLOR, colorQuantization);
+            encoder.SetAttributeQuantization(encoderModule.GENERIC, skinQuantization);
+
+            const encodedLen = encoder.EncodeMeshToDracoBuffer(newMesh, encodedDracoDataArray);
+            if (encodedLen > 0) {
+              console.log("Encoded size is " + encodedLen);
+            } else {
+              console.log("Error: Encoding failed.");
```

Comment:
These `console.log`s should be removed as well.

Reply:
Done.